components/applications-service/pkg/storage/postgres/service_groups.go

```diff
@@ -337,6 +352,14 @@ func queryFromStatusFilter(text string) (string, error) {
 	}
 }
 
+func queryFromFieldFilter(field string, envs []string) (string, error) {
+	formattedVals := strings.Join(envs[:], "','")
```

Comment:
do we need to restrict the characters we allow here to prevent SQL nastiness or does that happen elsewhere?